crypto/evp/evp_fetch.c

```diff
@@ -105,10 +170,13 @@ static void *get_method_from_store(OPENSSL_CTX *libctx, void *store,
      */
     if ((name_id = methdata->name_id) == 0) {
         OSSL_NAMEMAP *namemap = ossl_namemap_stored(libctx);
+        const char *names = methdata->names;
+        const char *q = strchr(names, NAME_SEPARATOR);
+        size_t l = q == NULL ? strlen(names) : (size_t)(q - names);
```

Comment:
maybe use the same notation as above l = (q == NULL ? ...);

Reply:
Ah, yeah, sure